src/style-toaster.scss

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2023 Alaska Airlines. All right reserved. Licensed under the Apache-2.0 license
+// See LICENSE in the project root for license information.
+
+// ---------------------------------------------------------------------
+
+// Import Auro tokens
+@import './../node_modules/@alaskaairux/design-tokens/dist/tokens/SCSSVariables';
+@import './../node_modules/@alaskaairux/design-tokens/dist/tokens/SassCustomProperties';
+
+@import './../node_modules/@aurodesignsystem/webcorestylesheets/dist/breakpoints';
+@import './../node_modules/@aurodesignsystem/webcorestylesheets/dist/core';
+@import './../node_modules/@aurodesignsystem/webcorestylesheets/dist/utilityClasses/displayProperties';
+
+// Support for auroElement styles
+// @import './node_modules/@aurodesignsystem/webcorestylesheets/dist/auroElement/auroElement';
+
+// container styles - defines the look of the outer custom component
+:host {
+  position: fixed;
+  z-index: var(--auro-depth-modal);
+  right: 2%;
+  bottom: 2%;
+  left: 2%;
+  display: flex;
+  flex-direction: column-reverse;
+  align-items: flex-end;
+  gap: var(--auro-size-200);
+  pointer-events: none;
+  
+  @media ($auro-grid-breakpoint-sm <=width < $auro-grid-breakpoint-lg) {
+    right: 1%;
+    left: unset;
+    width: 100%;
+}
+  @media (width >=$auro-grid-breakpoint-lg) {
+    right: 1%;
+    left: unset;
+    width: 100%;
+}
+
+}
+
+// Selector for native :focus-visible support
+:focus-visible {
```

Comment:
what element is this selector for? I'd feel more comfortable with a less vague selector. Encapsulation makes it not really a big deal but could trip up future work in this component

Reply:
This selector should NOT be used. This was for a [polyfill](https://github.com/WICG/focus-visible) we no longer support and is not installed in this project. 

Please refer to the current [pseudo-class](https://css-tricks.com/almanac/selectors/f/focus-visible/) spec.